autogpt_platform/backend/backend/data/queue.py

```diff
@@ -36,26 +29,23 @@ def get(self) -> ExecutionResult | None:
 
 class RedisEventQueue(AbstractEventQueue):
     def __init__(self):
-        self.connection = None
         self.queue_name = redis.QUEUE_NAME
 
-    def connect(self):
-        self.connection = redis.connect()
+    @cached_property
```

Comment:
seems redundant since `redis.get_redis()` already returns a "cached" reference from `redis`

Reply:
Changed to @property